packages/ui/src/i18n/dictionaries/authenticator/ja.ts

```diff
@@ -22,7 +22,7 @@ export const jaDict: AuthenticatorDictionary = {
   'Enter your code': 'コードを入力',
   'Enter your Email': 'メールアドレスを入力',
   'Enter your phone number': '電話番号を入力',
-  'Enter your username': 'ユーザー名を入力 ',
+  'Enter your Username': 'ユーザー名を入力 ',
```

Comment:
@esauerbo Can we confirm that we don't use both lowercase and uppercase versions of this string?